tools/sweep_eval.py

```
@@ -131,11 +131,11 @@ def main(cfg: DictConfig | ListConfig) -> int:
         # Add policy to wandb sweep
         policy_store.add_to_wandb_sweep(cfg.sweep_name, policy_pr)
 
-        # Record observation in CARBS if enabled
+        # Record observation in Protein optimizer
         total_time = train_time + eval_time
         logger.info(f"Evaluation Metric: {eval_metric}, Total Time: {total_time}")
 
-        WandbCarbs._record_observation(wandb_run, eval_metric, total_time, allow_update=True)
+        MettaProtein._record_observation(wandb_run, eval_metric or 0.0, total_time, allow_update=True)
```

why would eval_metric be none? i think we should crash rather than silently succeed.

---

Added a guard.